controllers/scriptStorage.js

```diff
@@ -156,7 +162,9 @@ exports.sendScript = function (aReq, aRes, aNext) {
     ++aScript.installs;
     ++aScript.installsSinceUpdate;
 
-    aScript.save(function (aErr, aScript) { });
+    aScript.save(function (aErr, aScript) {
+      // WARNING: Resaving the script... WHY?
```

Comment:
Ahh good point that I missed... different train of thought when STYLEGUIDE'ing... thanks.